### PR TITLE
contracts: remove parameter `chain`

### PR DESCRIPTION
**Pull request by xhjkl**
Parameter `chain` removed from everywhere as redundant.

### CODE REVIEW

File: website/data-storage/src/programs/programs.service.ts [hunk 1]
```diff
@@ -75,8 +74,8 @@ export class ProgramsService {
   }
 
   async findProgram(params: FindProgramParams): Promise<Program> {
-    const { id, chain, genesis, owner } = params;
-    const where = owner ? { id, chain, genesis, owner } : { id, chain, genesis };
+    const { id, genesis, owner } = params;
+    const where = owner != null ? { id, genesis, owner } : { id, genesis };
```
Comment (osipov-mit) [inline website/data-storage/src/programs/programs.service.ts:78]: Why is `!= null` necessary?

File: website/data-storage/src/programs/programs.service.ts [hunk 2]
```diff
@@ -88,11 +87,11 @@ export class ProgramsService {
     }
   }
 
-  async setStatus(id: string, chain: string, genesis: string, status: InitStatus): Promise<Program> {
+  async setStatus(id: string, genesis: string, status: InitStatus): Promise<Program> {
     return new Promise((resolve) => {
       setTimeout(async () => {
-        console.log(id, chain, genesis, status);
-        let program = await this.findProgram({ id, chain, genesis });
+        console.log(id, genesis, status);
```
Comment (osipov-mit) [inline website/data-storage/src/programs/programs.service.ts:93]: This string is unnecessary, we can delete it

**Review by osipov-mit — CHANGES_REQUESTED** [review]
Also you can delete `chain` from events-listener service. And from all entities in data-storage service.

File: website/data-storage/src/programs/programs.service.ts (hunk 2 above)
Comment (xhjkl) [reply website/data-storage/src/programs/programs.service.ts]: Done.

**Review by osipov-mit — APPROVED** [review]

**Pull request merged**